### PR TITLE
new spelling

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: docs/how_to/create_custom_map.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ Here's an example of creating a header from some generic data and an `astropy.co
     >>> from sunpy.coordinates import frames
     >>> from sunpy.map.header_helper import make_fitswcs_header
 
-    >>> data = np.arange(0, 100).reshape(10, 10)
+    >>> data = np.arrange(0, 100).reshape(10, 10)
```
Comment (samaloney) [inline docs/how_to/create_custom_map.rst:29]: Fail

File: sunpy/coordinates/spice.py [hunk 2]
```diff
@@ -167,7 +167,7 @@ def _convert_to_et(time):
 
 def _astropy_frame_name(spice_frame_name):
     # Replace plus/minus characters in the SPICE frame name with lowercase 'p'/'n'
-    return f"spice_{spice_frame_name.translate(str.maketrans('+-', 'pn'))}"
+    return f"spice_{spice_frame_name.translate(str.maketrans('+-', 'on'))}"
```
Comment (samaloney) [inline sunpy/coordinates/spice.py:170]: fail

File: sunpy/conftest.py [hunk 3]
```diff
@@ -128,11 +128,11 @@ def sunpy_test_run(request):
...
 def hide_parfive_progress(request):
     """
-    Set the PARFIVE_HIDE_PROGESS to hide the parfive progress bar in tests.
+    Set the PARFIVE_HIDE_PROGRESS to hide the parfive progress bar in tests.
     """
-    os.environ["PARFIVE_HIDE_PROGESS"] = "True"
+    os.environ["PARFIVE_HIDE_PROGRESS"] = "True"
     yield
-    del os.environ["PARFIVE_HIDE_PROGESS"]
+    del os.environ["PARFIVE_HIDE_PROGRESS"]
```
Comment (alasdairwilson) [inline sunpy/conftest.py:135]: errrrrm

File: sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py [hunk 4]
```diff
@@ -93,7 +93,7 @@ def test_client_repr(LCClient):
     Repr check
     """
     output = str(LCClient)
-    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClien'
+    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClient'
```
Comment (samaloney) [inline sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py:96]: Unsure with the :50 it might end early?

Comment (alasdairwilson) [inline sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py:96]: interesting choice tbh to do 50 and not 51

**Review by samaloney — CHANGES_REQUESTED** [review]

File: sunpy/net/dataretriever/sources/tests/test_rhessi.py [hunk 5]
```diff
@@ -188,7 +188,7 @@ def test_client_repr(LCClient):
     Repr check
     """
     output = str(LCClient)
-    assert output[:50] == 'sunpy.net.dataretriever.sources.rhessi.RHESSIClien'
+    assert output[:50] == 'sunpy.net.dataretriever.sources.rhessi.RHESSIClient'
```
Comment (alasdairwilson) [inline sunpy/net/dataretriever/sources/tests/test_rhessi.py:191]: same here

File: _typos.toml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+default.extend-ignore-identifiers-re = [
+    "AN.*ed", # ANDed
+    ".*np.arange.*",
```
Comment (alasdairwilson) [inline _typos.toml:3]: is the extra . here breaking it?

File: sunpy/coordinates/spice.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nabobalis) [reply sunpy/coordinates/spice.py:170]: Damn it, I forgot to add that check.

File: sunpy/conftest.py (hunk 3 above)
Comment (nabobalis) [reply sunpy/conftest.py:135]: We have misspelt this either here for ages or parfive has.

File: sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py (hunk 4 above)
Comment (nabobalis) [reply sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py:96]: Oh I should add :51 to make sure its ok.

Comment (nabobalis) [reply sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py:96]: I did this as I was lazy and like the number 50.

File: _typos.toml (hunk 6 above)
Comment (nabobalis) [reply _typos.toml:3]: Oh yeah

File: sunpy/conftest.py (hunk 3 above)
Comment (ayshih) [reply sunpy/conftest.py:135]: I sent the following message on Element on 2022-Oct-20:
> @Cadair: Um... As far as I can tell, parfive checks the environment variable PARFIVE_HIDE_PROGRESS (https://github.com/Cadair/parfive/blob/c75353cee89715a6eb1bfd9dce704a13cbd0ba52/parfive/config.py#L59), but no external code appears to set the correct environment variable. Instead, they have the misspelled PARFIVE_HIDE_PROGESS (https://github.com/search?q=PARFIVE_HIDE_PROGESS&type=code, and this misspelling is in the parfive docs) or HIDE_PARFIVE_PROGESS (https://github.com/search?q=HIDE_PARFIVE_PROGESS&type=code). Amusingly, sunpy core uses both misspellings.

**Review by wtbarnes — APPROVED** [review]

**Pull request merged**